src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/processor/ZipVerifiers.java

```diff
@@ -91,23 +92,10 @@ static ZipInputStream noOpVerification(ZipStreamWithSignature zipWithSignature)
     static ZipInputStream sha256WithRsaVerification(ZipStreamWithSignature zipWithSignature)
         throws DocSignatureFailureException {
         Map<String, byte[]> zipEntries = extractZipEntries(zipWithSignature.zipInputStream);
-        if (!verifyFileNames(zipEntries)) {
-            log.warn("Signature Failure. Zip entries do not match expected file names. "
```

Comment:
These warnings are gone, but they are unnecessary - the information is logged at the task level.